tests/Resources/Page/ODH/JupyterHub/HighAvailability.robot

```diff
@@ -35,6 +37,17 @@ Verify JupyterHub Deployment
 
     FOR  ${index}  IN RANGE  0  ${nPods}
         &{pod} =  Set Variable  ${component}[${index}]
+        ${version-check} =  Is RHODS Version Greater Or Equal Than  1.9.0
+        IF  ${version-check}==True
+            # Grab x.y.z version of jupyterhub
+            ${jh_version} =    Run  oc -n redhat-ods-applications exec ${pod.metadata.name} -c jupyterhub -- pip show jupyterhub | grep Version: | awk '{split($0,a); print a[2]}'
```

Comment:
There is a keyword  `Verify Installed Library Version`  to check for an exact match. 

Maybe you could create a new `Verify Installed Library Version Is In Range`  (or similar name) with the logic implemented here

Reply:
I can but it doesn't seem something related to this PR. 
Here the logic is also slightly different than usual because `Verify Installed Library Version` uses `x.y` while this uses `x.y.z`